### ISSUE TITLE
HTML Progress value add/remove doesn't work

### ISSUE THREAD

**Issue by mattbdc** [vobyjs/voby#27]
https://codesandbox.io/s/voby-demo-clock-forked-z5c9sz?file=/src/index.tsx

Similar bug on Solidjs as 1835

I have a feeling I may not be using Voby correctly though but removing the () in JSX doesn't help either from my brief experiments

**Comment by fabiospampinato** [comment]
Very interesting edge case though, without special-casing this internally I'm not sure it could be supported by Voby 🤔 It's a very weird API this <progress> element has.

**Comment by fabiospampinato** [comment]
Something like this potentially works, the same workaround would work in Solid:

```js
import * as React from 'voby';
import {$, render, If} from 'voby';

const Clock = (): JSX.Element => {

  const x = $<number>(-1);

  const toggle = () => {
    x(xv => xv >= 0 ? -1 : 40)
  }

  return (
    <div class="clock">
      <p>{x}</p>
      <If when={() => x() >= 0} fallback={<progress />}>
        <progress value={x} />
      </If>
      <button onClick={toggle}>TOGGLE</button>
    </div>
  );

};

render ( <Clock />, document.body );
```

I guess I'll just have to special-case this for it to be supported properly.

**Comment by fabiospampinato** [comment]
It's doing [both](https://github.com/vobyjs/voby/blob/4b5c1eab5c30f39556d352152d604709cabfb8b3/src/utils/setters.ts#L744-L750), `<progress>` just happens to throw if it sees an undefined for some reason. 

I've [special-cased](https://github.com/vobyjs/voby/blob/4b5c1eab5c30f39556d352152d604709cabfb8b3/src/utils/setters.ts#L730-L733) this for `<progress>`, so `undefined` will work the same in the next release.

**Comment by fabiospampinato** [comment]
Fixed in v0.55, thanks for the bug report!
